salt/lax/bot-lax-adaptor.sls

```diff
@@ -80,3 +80,62 @@ logrotate-for-bot-lax-adaptor-logs:
     file.managed:
         - name: /etc/logrotate.d/bot-lax-adaptor
         - source: salt://lax/config/etc-logrotate.d-bot-lax-adaptor
+
+{% for fname in ['test.log', 'scrape.log', 'validate.log'] %}
```

Comment:
yes, `/var/log/bot-lax-adaptor/` perhaps, then I can just target the whole directory and allow them to be created as necessary